pkg/services/rpcsrv/subscription_test.go

```diff
@@ -261,6 +268,39 @@ func TestFilteredSubscriptions(t *testing.T) {
 				require.Equal(t, "my_pretty_notification", n)
 			},
 		},
+		"notification matching contract hash and parameter": {
+			params: `["notification_from_execution", {"contract":"` + testContractHash + `", "parameters":[{"type":"Any","value":null},{"type":"Hash160","value":"449fe8fbd4523072f5e3a4dfa17a494c119d4c08"}]}]`,
+			check: func(t *testing.T, resp *neorpc.Notification) {
+				rmap := resp.Payload[0].(map[string]any)
+				require.Equal(t, neorpc.NotificationEventID, resp.Event)
+				c := rmap["contract"].(string)
+				require.Equal(t, "0x"+testContractHash, c)
+				// it should be exact unique "Init" call sending all the tokens to the contract itself
```

Comment:
Comment should start with a capital letter and end with a period.

Reply:
i threat it more like this type of comments: https://github.com/nspcc-dev/neo-go/blob/ff15e39363ea44cbe337f493c434304de38b2382/cli/util/util_test.go#L52

also, if linter allows it, it is ok, IMO

fixed

Reply:
This type of comments continues the line of code before it. I should be more clear, I was talking about *stand-alone* comments.